CHANGELOG.md

```diff
@@ -39,6 +40,8 @@ All notable changes to this project will be documented in this file.
 - Provisioning config caching
 - Mailboxes not being synced due to short circuiting
 - Handle attachments without transfer encoding properly
+- Account settings modal close button
+- Set job time insensitive and lower interval for quota
```

Comment:
this was just the leftovers https://github.com/nextcloud/mail/pull/8655

quota depletion warning is the new feature. any modifications of the feature are not a *change* from a previous version, so they do not go into the changelog.